### ISSUE TITLE
Add support for Gradle

### ISSUE THREAD

**Issue by m-reza-rahman** [eclipse-ee4j/starter#94]
Likely a low priority feature. Maven is pretty standard for Jakarta EE projects.

**Comment by m-reza-rahman** [comment]
To start with, could simply be instructions on how to convert from Maven to Gradle.

**Comment by m-reza-rahman** [comment]
Personally I agree this is a low priority feature. Maven is pretty standard for Jakarta EE projects that I have seen. To start with, this could simply be instructions on how to convert from Maven to Gradle.

I do however think it is a bad idea to ignore this requirement for too long. This was one of the feedback items from the 1.0 release, along with the need for a Starter UI in addition to Archetypes. This is why this issue was initially created. All the other Starters including the Spring and MicroProfile ones have support for Gradle, so it would be odd for us to not prioritize this at all. Most reports (e.g. https://snyk.io/jvm-ecosystem-report-2021/) show approximately a third of developers using Gradle. This is not a population we can afford to outright ignore for too long.

**Comment by kito99** [comment]
I agree with @m-reza-rahman -- lots of folks use Gradle, so once we get the key features working with Maven, we should move on to Gradle sooner rather than later. We should also build things such that it's easy to add Gradle support later.

**Comment by hantsy** [comment]
When considering Gradle, we could select Kotlin or Groovy as the script language.

I like Gradle Kotlin script.

**Comment by m-reza-rahman** [comment]
At least initially, we’ll be using this: https://www.baeldung.com/maven-convert-to-gradle. It’s easy to add a UI option for either.
